### PR TITLE
pr for resource group

### PR DESCRIPTION
**Pull request by RithvickAR**

### CODE REVIEW

File: cmd/resourcegroup.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ package cmd
 import (
 	"fmt"
 
-	"github.com/github.com/RithvickAR/aksctl/coreaksctl"
+	"github.com/RithvickAR/aksctl/coreaksctl"
```
Comment (anoopl) [inline cmd/resourcegroup.go]: This should be : 	"github.com/RithvickAR/aksctl/coreaksctl"
	"github.com/adfolks/aksctl/coreaksctl"
Right? Please change all refernces

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module github.com/RithvickAR/aksctl
```
Comment (anoopl) [inline go.mod]: should be adfolks/aksctl

File: go.sum [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+google.golang.org/grpc v1.21.0/go.mod h1:oYelfM1adQP15Ek0mdvEgi9Df8B9CZIaU1084ijfRaM=
+gopkg.in/alecthomas/kingpin.v2 v2.2.6/go.mod h1:FMv+mEhP44yOT+4EoQTLFTRgOQ1FBLkstjWtayDeSgw=
+gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod h1:Co6ibVJAznAaIkqp8huTwlJQCZ016jof/cbN4VW5Yz0=
+gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127/go.mod h1:Co6ibVJAznAaIkqp8huTwlJQCZ016jof/cbN4VW5Yz0=
+gopkg.in/resty.v1 v1.12.0/go.mod h1:mDo4pnntr5jdWRML875a/NmxYqAlA73dVijT2AXvQQo=
+gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7/go.mod h1:JAlM8MvJe8wmxCU4Bli9HhUf9+ttbYbLASfIpnQbh74=
+gopkg.in/yaml.v2 v2.2.1/go.mod h1:hI93XBmqTisBFMUTm0b8Fm+jr3Dg1NNxqwp+5A1VGuI=
+gopkg.in/yaml.v2 v2.2.2 h1:ZCJp+EgiOT7lHqUV2J862kp8Qj64Jo6az82+3Td9dZw=
+gopkg.in/yaml.v2 v2.2.2/go.mod h1:hI93XBmqTisBFMUTm0b8Fm+jr3Dg1NNxqwp+5A1VGuI=
+honnef.co/go/tools v0.0.0-20190102054323-c2f93a96b099/go.mod h1:rf3lG4BRIbNafJWhAfAdb/ePZxsR/4RtNHQocxwk9r4=
```
Comment (anoopl) [inline go.sum]: Do we need this file on repo? Look at other repos and please figureout

**Review by anoopl — COMMENTED** [review]
Please see my comments

File: go.sum (hunk 3 above)
Comment (RithvickAR) [reply go.sum]: file removed

File: go.mod (hunk 2 above)
Comment (RithvickAR) [reply go.mod]: changed

File: cmd/resourcegroup.go (hunk 1 above)
Comment (RithvickAR) [reply cmd/resourcegroup.go]: changed

**Pull request closed without merging**